src/test/resources/logicalOperationTestData.json

```diff
@@ -0,0 +1,10 @@
+{"_time": "2001-01-01T01:01:01.010+03:00", "_raw": "\"raw 01\"", "index": "index_A", "sourcetype": "A:X:0", "source": "imfile:computer01.example.com:01.log", "host": "computer01.example.com", "partition": "hundred-year/2001/01-01/computer01.example.com/01/01.logGLOB-2001010101.log.gz", "offset": 1},
+{"_time": "2002-02-02T02:02:02.020+03:00", "_raw": "\"raw 02\"", "index": "index_A", "sourcetype": "A:Y:0", "source": "imfile:computer02.example.com:02.log", "host": "computer02.example.com", "partition": "hundred-year/2002/02-02/computer02.example.com/02/02.logGLOB-2002020201.log.gz", "offset": 2},
```

Comment:
this could be changed to be enclosed in [ ] to make it a valid JSON array
meaning add [ at the beginning of the file and ] at the end of the file.

Reply:
Will be worked in #338